### PR TITLE
added a body color for consistency.set colors in tailwind, added a cu…

### PR DESCRIPTION
**Pull request by ArielHolman**
added a body color for consistency.set colors in tailwind, added a custom ts file so we can use to declare module for svg, created svg for logo, completed template for nav.

### CODE REVIEW

File: src/styles/main.css [hunk 1]
```diff
@@ -1,3 +1,7 @@
+body{
+  background-color: #f6f4f4eb;
```
Comment (devholman) [inline src/styles/main.css]: can we use a token name for the color being used here?

File: tsconfig.json [hunk 2]
```diff
@@ -35,5 +35,5 @@
     "isolatedModules": true
   },
   "exclude": ["./dist/**/*", "./node_modules/**/*"],
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"]
+  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", "src/components", "src/custom.d.ts"]
```
Comment (devholman) [inline tsconfig.json]: I thought the blob `**.*/ts` would include `src/custom.d.ts` & `src/components`. Does it not?

**Review by devholman — CHANGES_REQUESTED** [review]

File: tsconfig.json (hunk 2 above)
Comment (ArielHolman) [reply tsconfig.json]: fixed.

File: src/styles/main.css (hunk 1 above)
Comment (ArielHolman) [reply src/styles/main.css]: Created CSS variable 🥇

**Review by devholman — APPROVED** [review]
LGTM

**Pull request merged**